Recipes/Recipes/ProfileModule/Presenter/ProfilePresenter.swift

```diff
@@ -0,0 +1,21 @@
+// ProfilePresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Презентер для профиля
+final class ProfilePresenter {
+    // MARK: - Public Properties
+
+    weak var profileCoordinator: ProfileCoordinator?
+
+    // MARK: - Private Properties
+
+    private weak var view: UIViewController?
```

Comment:
```suggestion
    private weak var view: UIViewController?
```
UIViewController это конкретика а нужна абстакция см у других